packages/components/molecules/f-mega-modal/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-mega-modal",
   "description": "Fozzie Mega Modal – A Vue.js modal component",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
A small one, but these aren't patch releases – they're minor code changes, so should bump the minor versions